plugins/analysis-kuromoji/build.gradle

```diff
@@ -33,6 +33,8 @@ restResources {
   }
 }
 
+integTest.enabled = false
```

Comment:
Why is this being disabled?

Reply:
We need a plan for ripping the creation of the `integTest` task out from the `PluginBuildPlugin`. Otherwise this is going to be necessary for any plugin project that uses only yaml tests. Or tweak our test conventions logic so it doesn't complain about this scenario.

Reply:
I found a need to implement a `javaRestTest` as part of converting these plugins, so (without hacks) can't finish my conversion to `yamlRestTest` .. Between `javaRestTest`, `yamlRestTest`, and `internalClusterTest` there is no need for `integTest` to exist anymore.  I plan to rip out `integTest` once all of these are implemented.  It won't take long (but my attention is split among a few things at the moment).